content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
++++
+
+This is a story about relationships, and how they are the currency that moves
+this world forward. It is a story about ii's relationship with Dan Kohn, and how
+his visionary leadership and friendship changed our world, and the open source
+community.
+
+## Ships, Servers, and Friends
+
+In 2015, Hippie Hacker consulted for [Chef.io](https://www.chef.io/), with one
+of his assignments involving the [Maritime Telecommunications
+Network](https://en.wikipedia.org/wiki/Maritime_Telecommunications_Network)
+(MTN). Through this assignment, Hippie met Aaron Crickenburger, who at the time
+worked for MTN Chief innovation officer Bob Wise.
+
+Aaron and Hippie did amazing things together, deploying hands-off,
+over-the-internet, network booting of servers that were then mananged by Chef
+software. They spent several weeks together making this happen and through this
+developed a friendship that, in the rippling effect of relationships, would
+influence the direction of Kubernetes today.
+
+<!-- NOTE:
+I am not sure where this anecdote fits into the story about Dan, Kubernetes, and ii.  
+In the interest of having a clear narrative, I edited it out.
+
+During this time Hippie was preparing a presentation about the work he was doing
+with the [Marine Reach](https://marinereach.com/) vessels. He asked Bob and his
+team if he could show them his presentation for some feedback from a live
+audience. The team really resonated with Hippie’s passion for the things he was
+involved in with Marine Reach. -->
+
+During the MTN project, Bob shipped servers to Hippie's home in Portland,
+Oregon, to help with the network-booting work. When Hippie later moved to
+Aotearoa New Zealand, he was unable to bring the servers with them. Hippie then
+asked bob if he had any money available to help relocate the servers to Aotearoa
+New Zealand. Bob introduced Hippie to someone who could help. This person was
+[Dan Kohn](https://en.wikipedia.org/wiki/Dan_Kohn). Bob introduced the two of
+them, telling Dan about Hippie's work and values. A relationship was formed, the
+servers were relocated, and are still used as part of the infrastructure for
+Hippie's vision.
+
+## Balena and Cross-Cloud
+
+Now in Aotearoa New Zealand, Hippie began doing work for [Balena](https://www.balena.io/)
+with [Denver Williams](https://github.com/denverwilliams). This technology
+enabled the "image pull and networking boot" in new, exciting ways. Inspired,
+Hippie set to use this technology with Kubernetes, by pulling down the images
+and installing [GitLab](https://about.gitlab.com) on top of it. He showed a demo
+of this to Dan who, impressed by this innovative approach to using Kubernetes,
+asked Hippie if he'd like to join the CNCF to help demonstrate what the
+orginzation is capable of. Hippie jumped in, helping to create a [Demo of CNCF
+technologies](https://github.com/cncf/demo/).
+
+While the demo did not get much traction, it was foundational in expoloring
```

Comment:
expoloring -> exploring

Reply:
Fixed as well, Thanks @castrojo for your sharp eyes.